src/sql/plot.py

```diff
@@ -283,7 +279,8 @@ def _min_max(con, table, column, with_=None):
 
     if with_:
         query = str(store.render(query, with_=with_))
-
+    if sql.connection.Connection.current:
```

Comment:
what happens if we remove this if statement? is there any case where there isn't an active connection and the user is able to execute this?

Reply:
Handled in this PR
https://github.com/ploomber/jupysql/pull/171

Let's merge that then we can remove here

Reply:
cool, I reviewed #171, let's merge that one first then